backend/requirements.txt

```diff
@@ -0,0 +1,8 @@
+click==6.7
+Flask==0.12.2
```

Comment:
Why can't we use the stable version v1? Although the question has probably lost relevance considering that there are other options besides Flask on the table now. Context: https://forum.uavcan.org/t/yukon-ui-rest-api/390/3?u=pavel.kirienko